R/processing.R

```diff
@@ -76,5 +81,9 @@ add_pkg_info <- function(topic_data, pkg_folder) {
   topic_data$package <- list(package = description$Package, version = description$Version)
   return(topic_data)
 }
-
+formatAuthor <- function(author) {
+  return(list(name = paste(author$given, author$family),
+              email = ifelse(is.null(author$email), NA, author$email),
```

Comment:
I think `ifelse()` is best not used here. Use something like this instead:

```R
if (is.null(author$email)) NA else author$email
```

(more info [here](http://stackoverflow.com/questions/17252905/else-if-vs-ifelse))